### DISCUSSION TITLE
Connecting Return Path to Domain

### DISCUSSION THREAD

**Discussion by mangoDzA** [postalserver/postal#1759] [Help with using Postal] [answered]
Whenever I try to connect my domain(lets say xyz.com) using the DNS Setup on the server running Postal, I get the error below:
`There is a CNAME record at psrp.xyz.com but it points to rp.postal.xyz.com which is incorrect. It should point to rp.postal.example.com. `
The said error occurs even though I have the DNS set up on my domain provider(Google Domain)'s page. Strangely enough, it says everything is working when I connect `psrp.xyz.com` to `rp.postal.example.com`. I honestly cannot tell if this is a bug or a mistake on my end. I also tried following the guide on the Doc, but that one seems completely different to the things I had to do. 
Listed below are some screenshots of my current setup:
[DNS Setup Page](https://free-robux.download/HE2qk7LR)
[Google Domains(hidden stuff is probably not related to this)](https://free-robux.download/modBGhTl)

Is there a way to fix or resolve this issue? Thanks in advance.

**Comment by willpower232** [comment, marked as answer]
the grey box says example.com which means you haven't updated the config file to include the real domain names

https://github.com/postalserver/postal/blob/ea10ae18f10ec243c9e30a540a3de4d853155838/config/postal.defaults.yml#L80-L87

If you have, then you need to restart Postal to apply the config file changes.

**Reply by mangoDzA** [reply]
That worked, thank you so much - I appreciate the help.
